### ISSUE TITLE
Upskill team in AWS

### ISSUE THREAD

**Issue by ojongerius** [FreeCodeCamp/open-api#141]
I'm not a 100% on what current familiarity level is of the AWS stack but I do know it can be a little daunting and there are a lot of moving parts. I'd suggest at least one of the paid members on the team is familiar with how things tie together.

@freeCodeCamp/open-api what is your preferred way with getting up to speed here, and what would be a good timing? Happy to create a screencast, or pair with folks, whatever works.

**Comment by raisedadead** [comment]
Count me in. I think a screencast would be ideal for future reference, but I am okay on hopping on a call as well, whichever is more comfortable to you.

@Bouncey your thoughts?

**Comment by Bouncey** [comment]
Please see this [tweet](https://twitter.com/BounceyTaylor/status/956314244030582784?s=19)

Still waiting for an answer

**Comment by raisedadead** [comment]
Replied. Don't you have my no on speed dial yet? :wink:

**Comment by Bouncey** [comment]
You are my hero, but I can't even remember what that was for now.

**Comment by Bouncey** [comment]
So, yes. I would be up for this.

**Comment by ojongerius** [comment]
Sweet, we'll do this once the dust has settled a little, but before you would receive alerts (or worse, not receive alerts) for production being on 🔥 

I guess @QuincyLarson would receive those calls too and need to be up to speed also?

**Comment by QuincyLarson** [comment]
@ojongerius Yes - I would definitely like to know. Worst case scenario I can SSH in and break the glass.

**Comment by ojongerius** [comment]
Indeed, what are you going to when you have no more SSH! 👻

Cool, I'll cook something up closer to go live, as things are pretty in flux at the moment.

**Comment by QuincyLarson** [comment]
> Indeed, what are you going to when you have no more SSH! 👻

When we don't have any more servers, and everything's running on Lambda or a managed cluster? Take a day of vacation 😎

> Cool, I'll cook something up closer to go live, as things are pretty in flux at the moment.

OK - sounds good. Thanks for keeping us posted on this.

**Comment by imcodingideas** [comment]
@Bouncey I'm studying for the AWS Solutions Architect right now. What you are looking for is: 

https://aws.amazon.com/elasticloadbalancing/
